src/commands/watch.js

```diff
@@ -19,8 +19,7 @@ export default asyncCommand({
 		},
 		host: {
 			description: 'Hostname to start a server on',
-			default: '0.0.0.0',
-			alias: 'h'
```

Comment:
and this is supposed to be a separate PR, but since already a PR is in queue, it just appended !! 

Sorry for the mess 😅